src/split-panel/styles.scss

```diff
@@ -125,26 +125,28 @@ $app-layout-drawer-width: calc(#{awsui.$space-layout-toggle-diameter} + 2 * #{aw
 
 .slider-side {
   cursor: ew-resize;
-  margin: 0 0 0 2px;
+  margin-block: 0;
```

Comment:
Same to my previous comment, looking at the codebase there is a lot of instances in which we transform `margin: x y z f` into separate `margin-block-[start|end]` and `margin-inline-[start|end]`, would be nice if we kept it as consistent as it was, wdyt?